mayastor/src/bdev/nexus/nexus_io.rs

```diff
@@ -88,10 +105,16 @@ impl Bio {
     }
 
     /// create and return a new, initialized Bio object
-    pub fn new(pio: *mut spdk_bdev_io, in_flight: i8) -> Bio {
+    pub fn new(
```

Comment:
I think a name like `bio_prep()` would sounce better, would you agree? new to me sounds a lot like an actual new IO, its not really...

Reply:
Ok, I can change that.